modules/programs/alot.nix

```diff
@@ -142,17 +151,36 @@ in
       '';
     };
 
+    extraConfigStructured = mkOption {
+      type = extraConfigType;
+      default = {
+        initial_command = "search tag:inbox AND NOT tag:killed";
+        auto_remove_unread = true;
+        ask_subject = false;
+        handle_mouse = true;
+        input_timeout = 0.3;
+        prefer_plaintext = true;
+        thread_indent_replies = 4;
+      };
+      example = literalExample ''
```

Comment:
No need to put an example that exactly matches the default since they both show up in the documentation.